plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -6,6 +6,7 @@
  */
 
 use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\Enums\ProductStatus;
```

Comment:
This import seems to be unused.

Reply:
Right! Removed it in https://github.com/woocommerce/woocommerce/pull/53938/commits/bbd743cd3095e8aab422560fc377865bc9e7e7eb